cypress/integration/plugins/custom-import-map-dashboards/documentsLayer.spec.js

```diff
@@ -27,9 +27,9 @@ describe('Documents layer', () => {
     cy.contains('Select data source', { timeout: 60000 }).click({
       force: true,
     });
-    cy.contains('opensearch_dashboards_sample_data_flights').click();
+    cy.wait(5000).contains('opensearch_dashboards_sample_data_flights').click();
```

Comment:
Could you quickly check if you really need `wait` and `contains`?
https://docs.cypress.io/guides/references/best-practices#Unnecessary-Waiting

You can do it as a follow up task.

Reply:
Feel free to merge or decide to research a bit in this PR. I am ok either way. @junqiu-lei

Reply:
Yes, currently we need this "wait", this above line code backend is querying data from OpenSearch, it may take some time, so I add wait here. But yeah later I can try to find a better way to replace here. Thanks!